app/containers/ReactionsList.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { FlatList, StyleSheet, Text, Pressable, View } from 'react-native';
+import ScrollableTabView from 'react-native-scrollable-tab-view';
+
+import Emoji from './message/Emoji';
+import { useTheme } from '../theme';
+import { TGetCustomEmoji } from '../definitions/IEmoji';
+import { IReaction } from '../definitions';
+import Avatar from './Avatar';
+import sharedStyles from '../views/Styles';
+
+const styles = StyleSheet.create({
+	reactionsListContainer: { height: '100%', width: '100%' },
+	tabBarItem: {
+		paddingHorizontal: 10,
+		paddingBottom: 10,
+		justifyContent: 'center',
+		alignItems: 'center',
+		flexDirection: 'row'
+	},
+	reactionCount: { marginLeft: 5 },
+	tabLine: { position: 'absolute', left: 0, right: 0, bottom: 0 },
+	emojiName: { margin: 10 },
+	userItemContainer: { marginHorizontal: 10, marginVertical: 5, flexDirection: 'row' },
+	usernameContainer: { marginHorizontal: 10, justifyContent: 'center' },
+	usernameText: { fontSize: 17, ...sharedStyles.textMedium }
+});
+const standardEmojiStyle = { fontSize: 20, color: '#fff' };
+const customEmojiStyle = { width: 25, height: 25 };
+
+interface IReactionsListBase {
+	baseUrl: string;
+	getCustomEmoji: TGetCustomEmoji;
+}
+
+interface IReactionsListProps extends IReactionsListBase {
+	reactions?: IReaction[];
+}
+
+interface ITabBarItem extends IReactionsListBase {
+	tab: IReaction;
+	index: number;
+	activeTab?: number;
+	goToPage?: (index: number) => void;
+}
+interface IReactionsTabBar extends IReactionsListBase {
+	activeTab?: number;
+	tabs?: IReaction[];
+	goToPage?: (index: number) => void;
+}
+
+const TabBarItem = ({ tab, index, goToPage, activeTab, baseUrl, getCustomEmoji }: ITabBarItem) => {
+	const { colors } = useTheme();
+	return (
+		<Pressable
```

Comment:
Sure there would need to be a pressable here?
Wouldn't a TouchableOpacity already solve the problem?

Reply:
I was told in the above reviews to not use TouchableOpacity anymore https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4346#discussion_r920081989